buildstockbatch/postprocessing.py

```diff
@@ -218,17 +218,28 @@ def read_results_json(fs, filename):
 
 
 def read_enduse_timeseries_parquet(fs, filename, all_cols):
-    with fs.open(filename, 'rb') as f:
-        df = pd.read_parquet(f, engine='pyarrow')
-    building_id = int(re.search(r'bldg(\d+).parquet', filename).group(1))
-    df['building_id'] = building_id
-    for col in set(all_cols).difference(df.columns.values):
-        df[col] = np.nan
-    return df[all_cols]
-
-
-def read_and_concat_enduse_timeseries_parquet(fs, filenames, all_cols):
-    return pd.concat(read_enduse_timeseries_parquet(fs, filename, all_cols) for filename in filenames)
+    try:
+        with fs.open(filename, 'rb') as f:
+            df = pd.read_parquet(f, engine='pyarrow')
+        building_id = int(re.search(r'bldg(\d+).parquet', filename).group(1))
+        df['building_id'] = building_id
+        for col in set(all_cols).difference(df.columns.values):
+            df[col] = np.nan
+        return df[all_cols]
+    except Exception:
+        return pd.DataFrame(columns=[all_cols]+['building_id'])
```

Comment:
This is a pretty broad `except` block. It looks like it just returns an empty dataframe with the appropriate columns if anything goes wrong. Intentionally creating these kinds of silent errors is troubling because it means that there could be major problems with reading some of the parquet files and the user will not know and think everything is fine.

Reply:
Yeah, agreed it could mask errors. But I think a few times there was an issue reading a handful of parquet files that resulted in crashing everything, so I thought it might be a good idea to assemble whatever can be assembled. Do you think it would be okay to have this try-except if we only wrap the read_parquet call?

Reply:
I figured that's why you had this here. I don't know if it functionally matters much to only wrap the read_parquet call rather than do this. My primary concern is that this will silently omit data the user is expecting. That could mess up results. Right now at least the user knows something went wrong because they don't get their results. Could we put some sort of error message in the postprocessing (or other relevant) log file saying, "building_id = x, upgrade = y failed to load parquet"? They still might miss that, but it would be documented somewhere. It would be better than what we have now, which is an obtuse error with very little to go on to find the problem.

Reply:
@nmerket I started working on the solution you proposed of logging the buildings we failed to read the timeseries, but then realized we would have to do it at multiple places. If there is any corrupted parquet it would fail here: https://github.com/NREL/buildstockbatch/blob/1cd0349759324e93348ce91d1c3ee59e683af8f3/buildstockbatch/postprocessing.py#L281

And, the corrupted parquet (or not being able to read from disk) is something I have seen only a couple of times when the eagle was having an issue, so it may not be worthwhile to account for that. Even if we need to handle that, maybe not in this PR. So, I have removed all such handling.